app/src/main/java/ai/elimu/appstore/MainActivity.java

```diff
@@ -60,24 +59,23 @@ protected void onStart() {
 
 
         // Ask for license number (used in custom projects)
-        SharedPreferences sharedPreferences = PreferenceManager.getDefaultSharedPreferences(this);
-        String licenseOption = sharedPreferences.getString(LicenseOptionActivity.PREF_LICENSE_OPTION, null);
+        String licenseOption = AppPrefs.getLicenseOption();
         Timber.i("licenseOption: " + licenseOption);
         if (TextUtils.isEmpty(licenseOption)) {
             Intent intent = new Intent(this, LicenseOptionActivity.class);
             startActivity(intent);
             finish();
         } else if ("no".equals(licenseOption)) {
             // Ask for locale (only apps for the selected locale will be downloaded)
-            String localeAsString = sharedPreferences.getString(LocaleActivity.PREF_LOCALE, null);
+            String localeAsString = AppPrefs.getLocale();
             Timber.i("localeAsString: " + localeAsString);
             if (TextUtils.isEmpty(localeAsString)) {
                 Intent intent = new Intent(this, LocaleActivity.class);
                 startActivity(intent);
                 finish();
             } else {
                 // Register device
-                boolean isRegistered = sharedPreferences.getBoolean(DeviceRegistrationActivity.PREF_IS_REGISTERED, false);
+                boolean isRegistered = AppPrefs.getRegisterStatus();
```

Comment:
If the name of the method is `getRegisterStatus()`, I do not expect to get a boolean in return. Can you rename the method?